packages/nimble-blazor/build/copyResources.js

```diff
@@ -2,32 +2,36 @@ const path = require('path');
 const glob = require('glob');
 const fs = require('fs');
 
-const destinationDirectory = path.resolve(__dirname, '../NimbleBlazor/wwwroot');
-console.log(`Destination directory for blazor assets: "${destinationDirectory}"`);
+copyEverythingToDestination(process.argv[2], process.argv[3]);
```

Comment:
Can we either pass an enum configuration of what to build with hard coded paths or not do sub encodings, i.e. the package name but just the part after the org / the project name but we assume a specifiic folder layout.

For build scripts I'd prefer to keep it exact and simple and just do an enum / string

```
const target = process.argv[2];
console.log('selected target: ',  target);
if (target === 'nimble-blazor') {do hard coded paths}
else if (target === 'spright-blazor') {do hard coded paths}
else {throw new Error()}
```

Limiting the parameterization of build scripts makes them a lot easier to troubleshoot